compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/repro-mutate-result-of-function-call-with-frozen-argument-in-function-expression.expect.md

```diff
@@ -0,0 +1,77 @@
+
+## Input
+
+```javascript
+import {identity, makeObject_Primitives, Stringify} from 'shared-runtime';
+
+function Example(props) {
+  const object = props.object;
+  const f = () => {
+    // The argument maybe-aliases into the return
+    const obj = identity(object);
+    obj.property = props.value;
+    return obj;
```

Comment:
Having invalid-react fixtures feels a little weird, since compiler semantics is technically UB.

I see what you're trying to express here though!